src/gui/CMakeLists.txt

```diff
@@ -210,6 +210,21 @@ if (DBUS)
         qtnotify/notifications.cpp
     )
     target_link_libraries(qbt_gui PRIVATE Qt5::DBus)
+
+    find_package(X11)
```

Comment:
The X11 package is probably not required. I was mislead by the source file names.

Reply:
@zeule 
So it would seem. I looked at the files and they only `#include` Qt stuff.

Reply:
>The X11 package is probably not required. I was mislead by the source file names.

Source file names were probably borrowed when I ported this feature from other soft (I don't remember from which one). It looks like they should have a different suffix (`dbus`).

Reply:
Fixed.

@glassez 
Also fixed and include that was causing the build to fail: https://github.com/qbittorrent/qBittorrent/pull/12746/commits/5a87306617372aea941dd2e1338bcbf90bc388d3

Backport here: https://github.com/qbittorrent/qBittorrent/pull/12790